core/test-extension/runtime/pom.xml

```diff
@@ -53,6 +53,15 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core-runtime</artifactId>
         </dependency>
+        <!-- Don't rely on JDK JAXB so code compiles and runs under both JDK8/11 -->
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+        </dependency>
```

Comment:
I think it should be done in the `quarkus-jaxb` extension and you should depend on it.

Reply:
I would be concerned the core module is depending on too many modules outside of the core?

Reply:
what's the resolution here?  There's lots of good stuff in this PR, I'd rather merge it than wait.

Is it important we have the quarkus-jaxb module if this jaxb dep is only used for testing? @gsmet could you open a separate issue if you think so? I'll merge soon if I don't see blockers :)

Reply:
Right, I would rather get the content out and then fix the dependency issue for the next release once I can verify that it does not introduce the issues like the native image build dependency caused. I created an issue for this. #1819